glyphs2fontir/src/source.rs

```diff
@@ -136,9 +137,10 @@ impl Source for GlyphsIrSource {
 
         let mut work: Vec<Box<IrWork>> = Vec::new();
         for glyph_name in glyph_names {
-            work.push(Box::from(
-                self.create_work_for_one_glyph(glyph_name, cache.font_info.clone())?,
-            ));
+            work.push(Box::from(self.create_work_for_one_glyph(
```

Comment:
`Box::from` is fine but `Box::new` is a bit more idiomatic.

Reply:
Huh I'd have sworn I got a red underliine when I guessed Box::new and then thought I was supposed to use from. Will fix, ty.

Reply:
Evidently I exclusively use Box::from. Fixed.